src/main/resources/META-INF/resources/primefaces/core/js.cookie.d.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Namespace for the cookie Javascript Cookie, available as `Cookies.set` and `Cookies.get`.
+ * 
+ * Contains some additional types and interfaces required for the typings.
+ * https://github.com/js-cookie/js-cookie
+ */
+declare namespace CookieJs {
```

Comment:
@melloware  The JSDocs basically look good to me. If it compiles with the `-P jsdoc` profile, it should be fine.

One minor thing though: `js-cookie` has semi-official type declarations [on DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/js-cookie). Currently, these are for version 2.3 (as js-cookie has only a release candidate of version 3 so far). Eventually we should just use that (installed via npm during the build process).

In preparation for that, it would be good to rename the namespace and interface so that it matches what's on DefinitelyTyped:

```typescript
declare namespace Cookies {
	/**
	 * Represents the properties of a cookie, other that its name and value.
	 */
	export interface CookieAttributes {
        }
}
```

(and change the parts in core.js accordingly)